src/python_testing/TC_DRLK_2_13.py

```diff
@@ -0,0 +1,656 @@
+        except InteractionModelError as e:
+            asserts.assert_equal(e.status, expected_status, f"Unexpected error returned: {e}")
+
+    async def send_clear_aliro_reader_config_cmd(self, expected_status: Status = Status.Success):
+        try:
+            await self.send_single_cmd(cmd=Clusters.DoorLock.Commands.ClearAliroReaderConfig(), endpoint=1,
+                                       timedRequestTimeoutMs=1000)
+            asserts.assert_equal(expected_status, Status.Success)
+        except InteractionModelError as e:
+            asserts.assert_equal(e.status, expected_status, f"Unexpected error returned: {e}")
+
+    async def send_set_aliro_reader_config_cmd(self, steps: dict, expected_status: Status = Status.Success, ):
+        try:
+            if self.pics_guard(self.check_pics("DRLK.S.F0d") and not self.check_pics("DRLK.S.F0e") and self.check_pics(
+                    "DRLK.S.C28.Rsp")) and steps.get("condition") == 1:
+                if steps.get("step") is not None:
+                    self.step(steps.get("step"))
+                await self.send_single_cmd(cmd=Clusters.DoorLock.Commands.SetAliroReaderConfig(
+                    signingKey=self.signingKey,
+                    verificationKey=self.verificationKey,
+                    groupIdentifier=self.groupIdentifier),
+                    endpoint=1,
+                    timedRequestTimeoutMs=1000)
+                self.skip_step(steps.get("skip_step"))
+                asserts.assert_equal(expected_status, Status.Success)
+            elif (self.pics_guard(self.check_pics("DRLK.S.F0e") and self.check_pics("DRLK.S.C28.Rsp"))
+                  and steps.get("condition") == 2):
+                self.skip_step(steps.get("skip_step"))
+                if steps.get("step") is not None:
+                    self.step(steps.get("step"))
+                await self.send_single_cmd(cmd=Clusters.DoorLock.Commands.SetAliroReaderConfig(
+                    signingKey=self.signingKey,
+                    verificationKey=self.verificationKey,
+                    groupIdentifier=self.groupIdentifier,
+                    groupResolvingKey=self.groupResolvingKey),
+                    endpoint=1,
+                    timedRequestTimeoutMs=1000)
+                asserts.assert_equal(expected_status, Status.Success)
+        except InteractionModelError as e:
+            logging.exception(f"Got exception when performing SetAliroReaderConfig {e}")
+            asserts.assert_equal(e.status, expected_status, f"Unexpected error returned: {e}")
+
+    async def get_credentials_status(self, credentialIndex: int, credentialType: cluster.Enums.CredentialTypeEnum,
+                                     step, userIndex, credential_exists=True):
+        if step is not None:
+            self.step(step)
+        try:
+
+            if self.pics_guard(self.check_pics("DRLK.S.F0d") and self.check_pics("DRLK.S.C24.Rsp") and self.check_pics(
```

Comment:
the condition can be placed outside the try to reduce indent. This looks to be like:

```py
flags =  ["DRLK.S.F0d", "DRLK.S.C24.Rsp", "DRLK.S.C25.Tx"]
if not self.pics_guard(all([self.check_picks(p) for p in flags])):
    return
```

Reply:
used similar code and execution is passing